services/auth/driving/web-mvc/src/main/java/nettee/auth/web/AuthCommandApi.java

```diff
@@ -129,17 +131,17 @@ public ResponseEntity<String> verifyEmail(@RequestBody EmailVerifyRequest reques
         return ResponseEntity.ok(token);
     }
 
-    @PostMapping("email/password/reset/send")
+    @PostMapping("/email/password/reset/send")
     @Operation(
             summary = "이메일 비밀번호 재설정 링크 전송",
             description = "사용자의 이메일로 비밀번호를 변경할 수 있는 링크를 발송합니다."
     )
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    public ResponseEntity<String> sendPasswordResetEmail(@RequestBody PasswordForgotRequest request) {
+    public void sendPasswordResetEmail(@RequestBody PasswordForgotRequest request) {
```

Comment:
두 가지 선택지가 있는 것 같습니다:

1. `@ResponseStatus(HttpStatus.NO_CONENT)`로 충분할 때  
    ```java
    // ResponseEntity와 중복되게 사용하지 않아도 됩니다.
    public void ...
    ```
2. ResponseEntity<Void> 사용 시  
    - @ResponseStatus(HttpStatus.NO_CONENT) 제거
    - .noContent() 등으로 반환

---

만약 코파일럿의 제안대로 기존 작업 방식처럼 `ResponseEntity`를 사용한다면
이처럼 `@ResponseStatus(HttpStatus.NO_CONENT)`를 제거할 수 있어 보입니다.

```suggestion
    @PostMapping("/email/password/reset/send")
    @Operation(
            summary = "이메일 비밀번호 재설정 링크 전송",
            description = "사용자의 이메일로 비밀번호를 변경할 수 있는 링크를 발송합니다."
    )
    @ResponseStatus(HttpStatus.NO_CONTENT)
    public ResponseEntity<Void> sendPasswordResetEmail(@RequestBody PasswordForgotRequest request) {
```